test/unit/relative-time/format.js

```diff
@@ -0,0 +1,56 @@
+define([
```

Comment:
For unit tests, having `test/unit/relative-time/format.js` is fine. Because, it maps to `src/relative-time/format`.